tendrl/commons/persistence/etcd_persister.py

```diff
@@ -1,14 +1,8 @@
-from tendrl.commons.etcdobj.etcdobj import Server as etcd_server
 from tendrl.commons.persistence.persister import Persister
 
 
 class EtcdPersister(Persister):
-    def __init(self, config):
-        super(EtcdPersister, self).__init__(config)
```

Comment:
The persister and its methods should be atomic in nature, eg: save_volume(volume) should only save the volume to etcd and disk.

The reason for not sending a config to a persister is to keep it sane and simple :)